pgkit/application/models/replica.py

```diff
@@ -50,10 +50,16 @@ def stop(self):
         execute_sync(f'pg_ctlcluster {self.version} {self.name} stop')
 
     def start(self):
-        execute_sync(f'pg_ctlcluster {self.version} {self.name} start', check_returncode=True)
+        if self.version >= 14: 
+            execute_sync(f'sudo -u postgres pg_ctlcluster {self.version} {self.name} start', check_returncode=True)
```

Comment:
Why need `sudo` for version > 14?

Reply:
pgkit should run in root mode. please remove sudo.

Reply:
sudo has been removed.